src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.AndroidLibraries.targets

```diff
@@ -39,6 +39,7 @@ projects.
       <AndroidJavaLibrary    Include="@(AndroidLibrary)" Condition=" '%(AndroidLibrary.Extension)' == '.jar' and '%(AndroidLibrary.Bind)' != 'true' " />
       <EmbeddedJar           Include="@(AndroidLibrary)" Condition=" '%(AndroidLibrary.Extension)' == '.jar' and '%(AndroidLibrary.Bind)' == 'true' " />
       <EmbeddedNativeLibrary Include="@(AndroidLibrary)" Condition=" '%(AndroidLibrary.Extension)' == '.so' " />
+      <ProguardConfiguration Include="@(AndroidLibrary)" Condition=" '%(AndroidLibrary.Extension)' == '.cfg' " />
```

Comment:
@jonathanpeppers I'm not sure what the default file filter should be under net 6. `*.cfg` or just `proguard.txt`.

Reply:
The defaults here are what split the `@(AndroidLibrary)` item group into the 6 other item groups.

I think you're wanting to make `*.cfg` automatically added to `@(ProguardConfiguration)`? I think we can do `**\*.cfg` and `**\proguard.txt` as wildcards in `AutoImports.props`?

Just keep in mind every .NET project type could import this file, so you must check `$(EnableDefaultAndroidItems)` for new `<ItemGroup/>` elements:

https://github.com/xamarin/xamarin-android/blob/6e3e3831afa958ab8126e268f7a128321f75dbd6/src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/Sdk/AutoImport.props#L19-L36